XenAdmin/Dialogs/OptionsPages/ExternalToolsOptionsPage.cs

```diff
@@ -60,32 +60,50 @@ public void Build()
 
         public bool IsValidToSave()
         {
-            if (radioButtonPutty.Checked && !File.Exists(textBoxPutty.Text))
+            // CA-362355: don't check if paths are valid if they haven't changed.

```

Comment:
Adding a comment here as the lines in question are not changed in this PR: I think it would be better if the control was populated from the Settings within the Build() method instead of the constructor.